From 907c07bb61d6f6d7e7b9ca131aba6728b90f09d3 Mon Sep 17 00:00:00 2001
From: Cosimo Cecchi <cosimoc@gnome.org>
Date: Thu, 28 Mar 2013 13:57:57 -0400
Subject: [PATCH] getting-started: fix parallel builds

We have to wait for the _HELP_C_PDFS target to be completed before
_HELP_PDF_OUTPUTS.
---
 m4/getting-started.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/getting-started.m4 b/m4/getting-started.m4
index 5224e6b..441522f 100644
--- a/m4/getting-started.m4
+++ b/m4/getting-started.m4
@@ -73,7 +73,7 @@ $(_HELP_C_PDFS): $(_HELP_C_FILES)
 	svgname=$(srcdir)/$(patsubst %.pdf,%.svg,[$]@); \
 	inkscape -z -A "[$]@" "$$svgname"
 
-$(_HELP_PDF_OUTPUTS): $(_HELP_LC_PDFS)
+$(_HELP_PDF_OUTPUTS): $(_HELP_LC_PDFS) $(_HELP_C_PDFS)
 	lc=`dirname [$]@`; \
 	files="$(foreach f,$(_HELP_PDFFILES),$$lc/$(f))"; \
 	pdfunite $$files "$$lc/$(_HELP_OUTPUT)"; \
-- 
1.8.1.4

